onnx2kerastl/elementwise_layers.py

```diff
@@ -27,7 +27,13 @@ def convert_elementwise_div(node, params, layers, lambda_func, node_name, keras_
 
     try:
         logger.debug('Divide numpy arrays.')
-        layers[node_name] = layers[node.input[0]] / layers[node.input[1]]
+        div = layers[node.input[0]] / layers[node.input[1]]
+        if input_0.dtype == input_1.dtype:
+            div = tf.cast(div, input_0.dtype)
```

Comment:
why should we cast if the dtypes are equal? isnt it the opposite casting if they are not equal?

Reply:
div op returns float for int inputs. in this case, we want to cast it to int, and then we get (x // y)

Reply:
so maybe instead of checking for `input_0.dtype == input_1.dtype` we can check for them being ints